### ISSUE TITLE
Attempted import error: 'createNamedContext' is not exported from 'consume-multiple-contexts'.

### ISSUE THREAD

**Issue by dnmahendra** [martpie/next-transpile-modules#41]
- [x] I HAVE READ [THE FAQ](https://github.com/martpie/next-transpile-modules#faq) AND MY PROBLEM WAS NOT DESCRIBED THERE

**Describe the bug**
I am trying to use `consume-multiple-contexts` node package. This package has issues in IE 11, as it is in es6.

I am using `next-transpile-modules` to transpile it. Following is my config - 

`babel.config.js`

```
module.exports = api => {
  api.cache(false)
  return {
    presets: [ 'next/babel' ],
    plugins: [
      [
        'styled-components',
        {
          displayName: true,
          ssr: true,
        },
      ],
      'lodash',
    ],
  }
}
```
`next.config.js`

```
const withCSS = require('@zeit/next-css')
const withBundleAnalyzer = require('@zeit/next-bundle-analyzer')
const withImages = require('next-images')
const publicRuntimeConfig = require('./libs/processEnv')
const withTM = require('next-transpile-modules')
const withPlugins = require('next-compose-plugins')

// next.js configuration
const nextConfig = {
  poweredByHeader: false,
  webpack: (config, {webpack}) => {
    // Fixes npm packages that depend on `fs` module
    config.node = {
      fs: 'empty',
    }

    const plugins = [
      new webpack.IgnorePlugin({
        resourceRegExp: /^\.\/locale$/,
        contextRegExp: /moment$/
      })
    ]

    config.plugins.push(...plugins);

    return config
  },
  publicRuntimeConfig
}

module.exports = withPlugins([
  withCSS,
  withImages,
  [withTM, {
    transpileModules: ['consume-multiple-contexts']
  }],
  [withBundleAnalyzer, {
    analyzeServer: [ 'server', 'both' ].includes(process.env.BUNDLE_ANALYZE),
    analyzeBrowser: [ 'browser', 'both' ].includes(process.env.BUNDLE_ANALYZE),
    bundleAnalyzerConfig: {
      server: {
        analyzerMode: 'static',
        reportFilename: '../bundles/server.html',
      },
      browser: {
        analyzerMode: 'static',
        reportFilename: '../bundles/client.html',
      },
    }
  }]
], nextConfig)

```
**Setup**

- Next.js version: 8.1.0
- `next-transpile-modules` version: 2.3.1
- Node.js version: 10.15.1
- `yarn` version: 1.13.0
- Operating System: MacOS

**Additional context**

**Comment by martpie** [comment]
Do you have a small repro I could play with?

**Comment by dnmahendra** [comment]
I will have to create one in codesandbox, but I am guessing this is the same problem with this issue as well - https://github.com/martpie/next-transpile-modules/issues/35. In general, any node module that uses es6 feature is broken in IE 11.

**Comment by martpie** [comment]
Mmmh i see, how do you usually polyfill for IE11?

**Comment by dnmahendra** [comment]
It's taking time for me to set up a Codesandbox so I have made a repo - https://github.com/dnmahendra/flash with the exact stage at which the project is. You can just clone and run `yarn install` and `yarn dev` to see the error. Regarding the IE11 thing nextjs claims to polyfill everything to support IE 11 but they don't transpile node modules. Which is where this library comes into the picture. I think I confused you bringing IE 11 into this, sorry for that. Now, using this library to transpile the problematic node module `consume-multiple-contexts` is not happening due to the error mentioned.

**Comment by dnmahendra** [comment]
I lifted these lines from your package and just updated my `next.conifg.js` webpack config and it works. 
```
    config.module.rules.push({
      test: /\.+(js|jsx)$/,
      loader: defaultLoaders.babel,
      include: [ /consume-multiple-contexts(?!.*node_modules)/ ]
    })
```

Maybe that gives you some idea of whats happening in the `next-transpile-modules` package.

**Comment by martpie** [comment]
Awesome thanks ! I'll try to have a look asap

**Comment by martpie** [comment]
Looks really similar to https://github.com/babel/babel/issues/8361

Not sure I can do anything here.

**Comment by martpie** [comment]
The upstream bug from npm since to be resolved since 6.10.3.

Feel free to comment if this needs to be re-opened.
